roachtest: upgrade/oldVersion=v2.1.3/nodes=5 failed

SHA: https://github.com/cockroachdb/cockroach/commits/d0e2f78183d2ce0a6b803127ee80143571c9cd4f

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=upgrade/oldVersion=v2.1.3/nodes=5 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1088865&tab=buildLog

```
The test failed on release-2.1:
	test.go:696,test.go:708,cluster.go:1090,upgrade.go:140,upgrade.go:263: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod start --encrypt=false teamcity-1088865-upgrade-oldversion-v2-1-3-nodes-5:1 returned:
		stderr:
		
		stdout:
		
		* 
		* Check out how to secure your cluster: https://www.cockroachlabs.com/docs/v2.1/secure-a-cluster.html
		*
		*
		* ERROR: cockroach server exited with error: failed to create engines: could not open rocksdb instance: Invalid argument: encryption was used on this store before, but no encryption flags specified. You need a CCL build and must fully specify the --enterprise-encryption flag
		*
		Failed running "start"
		E190111 13:25:27.836676 1 cli/error.go:230  exit status 1
		Error: exit status 1
		Failed running "start"
		
		github.com/cockroachdb/cockroach/pkg/cmd/roachprod/install.Cockroach.Start.func7
			/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/install/cockroach.go:396
		github.com/cockroachdb/cockroach/pkg/cmd/roachprod/install.(*SyncedCluster).Parallel.func1.1
			/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/install/cluster_synced.go:1301
		runtime.goexit
			/usr/local/go/src/runtime/asm_amd64.s:2361: 
		I190111 13:25:27.867150 1 cluster_synced.go:1383  command failed
		: exit status 1

```

I'm seeing this running it locally:

```
toblerone> E190111 14:47:59.023134 260 jobs/registry.go:302  error while adopting jobs: adopt-job: rpc error: code = Unimplemented desc = unknown service cockroach.sql.distsqlrun.DistSQL
```

cc @jordanlewis -- is this expected or an indication of a missing migration?

--- a/pkg/cmd/roachtest/upgrade.go
+++ b/pkg/cmd/roachtest/upgrade.go
@@ -137,7 +137,7 @@ func registerUpgrade(r *registry) {
 				t.Fatal(err)
 			}
 			c.Put(ctx, cockroach, "./cockroach", c.Node(i))
-			c.Start(ctx, t, c.Node(i), startArgsDontEncrypt)
+			c.Start(ctx, t, c.Node(i))
 			if err := sleep(stageDuration); err != nil {
 				t.Fatal(err)
 			}
@@ -159,7 +159,7 @@ func registerUpgrade(r *registry) {
 			t.Fatal(err)
 		}
 		c.Put(ctx, cockroach, "./cockroach", c.Node(nodes))
-		c.Start(ctx, t, c.Node(nodes), startArgsDontEncrypt)
+		c.Start(ctx, t, c.Node(nodes))
 		if err := sleep(stageDuration); err != nil {
 			t.Fatal(err)
 		}
@@ -200,7 +200,7 @@ func registerUpgrade(r *registry) {
 		}
 
 		// Restart the previously stopped node.
-		c.Start(ctx, t, c.Node(nodes-1), startArgsDontEncrypt)
+		c.Start(ctx, t, c.Node(nodes-1))
 		if err := sleep(stageDuration); err != nil {
 			t.Fatal(err)
 		}

